README.md

```diff
@@ -319,6 +319,35 @@ Saves the currently opened document in the viewer and returns the absolute path
 var path = await PdftronFlutter.saveDocument();
 ```
 
+### PdftronFlutter.commitTool()
+
+Commits the current tool.
+
+Only available for multi-stroke ink and poly-shape, and will return false for all other tools.
+
+```dart
+var committed = await PdftronFlutter.commitTool();
+print("Tool committed: $committed");
+```
+
+### PdftronFlutter.getPageCount()
+
+To get the current page count of the document.
```

Comment:
Instead:
Returns the total number of pages in the currently displayed document.